packages/react-scripts/scripts/eject.js

```diff
@@ -129,7 +129,7 @@ prompt(
   // Add Jest config
   console.log('  Adding ' + cyan('Jest') + ' configuration');
   appPackage.jest = createJestConfig(
-    filePath => path.join('<rootDir>', filePath),
+    filePath => path.posix.join('<rootDir>', filePath),
```

Comment:
I was looking at `"setupFiles": [ "<rootDir>\\config\\polyfills.js" ]` from #1417. Doesn't this mean the whole path needs to be fixed?